cvat/apps/engine/signals.py

```diff
@@ -28,6 +29,8 @@ def update_task_status(instance, **kwargs):
         status = StatusChoice.VALIDATION
 
     if status != db_task.status:
+        if status == StatusChoice.COMPLETED and not db_task.is_exchange_notified:
```

Comment:
set if after DB operations, because here could be a DB transaction

Reply:
I'm not sure I follow you. Here I have only one DB transaction (save).

Reply:
Not sure I follow you. I'm not doing db transactions inside notifier.